config.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
```

Comment:
I cant rebase because why have conflicts with the dependency in my branch and the branch master. Remember I modified the config.xml when run the Jenkins job . I think , my config.xml is the last one and the correct file